package.json

```diff
@@ -13,10 +13,14 @@
   "engines": {
     "node": "^12.20.0 || ^14.13.1 || >=16.0.0"
   },
-  "type": "module",
```

Comment:
This is now handled in subdirectory package.json files to establish type between module and commonjs (see https://github.com/iiroj/use-breakpoint/pull/19/files#r1205360438)